src/makeInfinityAdder.js

```diff
@@ -5,8 +5,10 @@
  * @return {function}
  */
 
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(x = 0) {
+  return (y) => {
+    return typeof y !== 'undefined' ? makeInfinityAdder(x + y) : x;
```

Comment:
you don't need typeof here
```suggestion
    return y !== undefined
      ? makeInfinityAdder(x + y)
      : x;
```